osp/core/session/db/db_wrapper_session.py

```diff
@@ -197,3 +201,23 @@ def _unreachable_warning(self, root_obj: Union[rdflib.URIRef, uuid.UUID]):
                              + " more" if len(unreachable) > 5
                         else "")
             logger.warning(warning)
+        if len(reachable) + len(unreachable) > 1000:
+            # Recommend disabling the warning for large datasets.
+            warning = "You are working with a large dataset. "
+            if not len(unreachable) > 0:
+                warning += "By default, when committing changes, " \
+                           "OSP-core looks for objects that are unreachable " \
+                           "from the wrapper object to generate a warning. " \
+                           "Generating such "
+            else:
+                warning += "Generating the above "
```

Comment:
Not sure that the if provides that much difference in terms of the text of the warning. Maybe the distinction could be clearer/removed?